recipes/log.c/all/conandata.yml

```diff
@@ -0,0 +1,5 @@
+sources:
+  "0.1.0":
```

Comment:
This should follow the virtual version outlined in the docs

Reply:
As defined:
`#define LOG_VERSION "0.1.0"`

Reply:
Unfortunately that does not lead to reproducible builds... what if there is a commit that fixes a bug but does not change the version?